metrics/pom.xml

```diff
@@ -44,10 +49,19 @@
             <groupId>com.expedia</groupId>
             <artifactId>metrics-java-jackson</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.expedia.adaptivealerting</groupId>
+            <artifactId>adaptive-alerting-kafka</artifactId>
+        </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>fluent-hc</artifactId>
+            <version>${apache.httpcomponents.version}</version>
```

Comment:
See above - manage dependencies from top POM please.

Reply:
Fixed.

Reply:
Please remove the `version` definitions from this file. They are automatically inherited.